padre_meddea/util/util.py

```diff
@@ -2,183 +2,10 @@
 This module provides general utility functions.
 """
 
-import os
-
-from astropy.time import Time
-
-import padre_meddea
-
-
-__all__ = ["create_science_filename", "parse_science_filename", "VALID_DATA_LEVELS"]
+__all__ = ["VALID_DATA_LEVELS"]
 
 TIME_FORMAT_L0 = "%Y%j-%H%M%S"
 TIME_FORMAT = "%Y%m%dT%H%M%S"
 VALID_DESCRIPTORS = ["eventlist", "spec-eventlist", "spec", "xraydirect"]
 VALID_DATA_LEVELS = ["l0", "l1", "l2", "l3", "l4"]
 FILENAME_EXTENSION = ".fits"
```

Comment:
Can these also be removed if we're using the swxsoc config?

Reply:
Yeah good catch on all of these!